pkgs/applications/science/physics/elmerfem/default.nix

```diff
@@ -32,15 +32,11 @@ stdenv.mkDerivation rec {
   "-DCMAKE_OpenGL_GL_PREFERENCE=GLVND"
   ];
 
-  patches = [
-    ./fix-cmake.patch
-  ];
-
   meta = with lib; {
     homepage = "https://elmerfem.org/";
     description = "A finite element software for multiphysical problems";
     platforms = platforms.unix;
-    maintainers = [ maintainers.wulfsta ];
+    maintainers = [ maintainers.wulfsta maintainers.broke ];
```

Comment:
If you want to add yourself as a maintainer change this to `with maintainers; [ broke wulfsta ];` to keep things ordered and less verbose.

Reply:
Thanks for the hint. The change is now included in the PR.